packages/@okta/vuepress-site/use_cases/inline_hooks/registration_hook/registration_hook/index.md

```diff
@@ -0,0 +1,244 @@
+
+Using the `com.okta.user.profile.update` commands you send in your response, you can modify the values of the attributes, or add values for other attributes, before the values are assigned to the Okta user profile created for the registering user.
+
+You can only set values for profile fields which already exist in your Okta user profile schema. Registration Inline Hook functionality can only set values, it cannot create new fields.
+
+> Note: The `password` field, along with any attributes that are marked as sensitive in your Okta user schema, are omitted from the information sent to your external service in the `data.userProfile` object. The password or its hash is never sent to your external service in any way.
+
+### data.action
+
+The action that Okta is currently set to take, regarding whether to allow this registration attempt.
+
+There are two possible values:
+
+- `ALLOW` indicates that the registration attempt will be allowed to proceed
+- `DENY` indicates that the registration attempt will be terminated (no user will be created in Okta)
+
+The action is `ALLOW` by default (in practice, `DENY` will never be sent to your external service).
+
+Using the `com.okta.action.update` [command](#supported-commands) in your response, you can change the action that Okta will take.
+
+## Objects in Response You Send
+
+For the Registration Inline Hook, the `commands` and `error` objects that you can return in the JSON payload of your response are defined as follows:
+
+### commands
+
+The `commands` object lets you invoke commands to modify or add values to the attributes in the Okta user profile that will be created for this user, as well as to control whether or not the registration attempt is allowed to proceed.
+
+This object is an array, allowing you to send multiple commands in your response. Each array element requires a `type` property and a `value` property. The `type` property is where you specify which of the supported commands you wish to execute, and `value` is where you supply parameters for that command.
+
+| Property | Description                                           | Data Type       |
+|----------|-------------------------------------------------------|-----------------|
+| type     | One of the [supported commands](#supported-commands). | String          |
+| value    | Operand to pass to the command.                       | [value](#value) |
+
+For example commands, see the [value](#value) section below.
+
+#### Supported Commands
+
+The following commands are supported for the Registration Inline Hook type:
+
+| Command                      | Description                                                       |
+|------------------------------|-------------------------------------------------------------------|
+| com.okta.user.profile.update | Set or modify an attribute in the Okta user profile that will be created for this user.                 |
```

Comment:
I think we should keep this consistent with the wording for Import Inline Hooks - `Change values of attributes in the user's Okta user profile.`

Reply:
Edited to use that wording.